app/controllers/state_file/questions/az_dependents_dob_controller.rb

```diff
@@ -0,0 +1,35 @@
+module StateFile
+  module Questions
+    class AzDependentsDobController < QuestionsController
+      layout "state_file/question"
+
+      def edit
+        @intake = current_intake
+        @dependents = @intake.dependents
+        @form = form_class.from_intake(@intake)
+      end
+
+      def update
+        @form = form_class.new(current_intake, form_params)
+        if @form.save && @form.valid?
+          redirect_to next_path
+        else
+          render :edit
+        end
```

Comment:
ditto this one: the whole thing can go because the parent implementation does all this